package.json

```diff
@@ -17,23 +24,29 @@
     "lint-staged": "^13.2.0",
     "marked": "^0.4.0",
     "prettier": "^2.8.4",
+    "publint": "^0.2.1",
```

Comment:
And once we drop the script, there's no value in having this as a dependency. Unless this is part of our CI process we can just run it manually when we want to check it
```suggestion
```